src/main/java/seedu/address/model/person/UniquePersonList.java

```diff
@@ -103,6 +105,15 @@ public void setPersons(List<? extends ReadOnlyPerson> persons) throws DuplicateP
         return FXCollections.unmodifiableObservableList(mappedList);
     }
 
+    /**
+     * Sorts person name in alphabetical order
+     */
+    public void sort() {
+        Comparator<Person> comparator = (p1, p2) -> (p1.getName().toString().compareTo(p2.getName().toString()));
+        Collections.sort(internalList, comparator);
```

Comment:
actually these 2 can combine into a line using the example I gave, no need collections.sort. but this is much better than before!